### PR TITLE
introduce POS sets with fewer points

### PR DESCRIPTION
**Pull request by RoyStegeman**

### CODE REVIEW

File: NNPDF_POS_F2C_40_17PTS/metadata.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+arxiv=
+description=DIS positivity for charm quark
+hepdata=
+data_url=
+nnpdf_id=POSF2C
```
Comment (andreab1997) [inline NNPDF_POS_F2C_40_17PTS/metadata.txt:5]: Should this change according to the yamldb?

File: NNPDF_POS_F2C_CCE_40/observable.yaml [hunk 2]
```diff
@@ -109,4 +109,13 @@ observables:
   - Q2: 5.0
     x: 0.66
     y: 1.0
+  - Q2: 5.0
+    x: 0.74
+    y: 1.0
```
Comment (andreab1997) [inline NNPDF_POS_F2C_CCE_40/observable.yaml:114]: Also, should we also have the version with fewer points here? Or at least we should change the name, no?

Comment (andreab1997) [reply NNPDF_POS_F2C_CCE_40/observable.yaml:114]: Done in [149b14d](https://github.com/NNPDF/pinecards/pull/176/commits/149b14d3d8508cbaa941af79f3d12bbbd9cdb848)

File: NNPDF_POS_F2C_40_17PTS/metadata.txt (hunk 1 above)
Comment (andreab1997) [reply NNPDF_POS_F2C_40_17PTS/metadata.txt]: Done in [149b14d](https://github.com/NNPDF/pinecards/pull/176/commits/149b14d3d8508cbaa941af79f3d12bbbd9cdb848)

**Comment by andreab1997** [comment]
@RoyStegeman have a look when you can, this should be ready to be merged

**Comment by RoyStegeman** [comment]
Thanks, this looks good

**Pull request merged**